auth/acquire.go

```diff
@@ -13,3 +13,9 @@ type Decorator interface {
 	// Decorate decorates the given http request with authorization header(s).
 	Decorate(ctx context.Context, req *http.Request) error
 }
+
+type DecoratorFunc func(context.Context, *http.Request) error
+
+func (f DecoratorFunc) Decorate(ctx context.Context, req *http.Request) error { return f(ctx, req) }
+
+var Nop = DecoratorFunc(func(context.Context, *http.Request) error { return nil })
```

Comment:
Made `Nop` public since it's used by our tests and I think it'll be useful for clients when they want to explicitly indicate their ancla client is not configured for auth.